idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -613,120 +641,131 @@ def run_priceseqfilter(input_fqs, uploader_start):
+    """FASTQ to FASTA conversion."""
+    fq2fa(input_fqs[0], result_path(FQ2FA_OUT1))
     if len(input_fqs) == 2:
-        fq2fa(input_fqs[1], os.path.join(RESULT_DIR, FQ2FA_OUT2))
-    write_to_log("finished job")
-    uploader_start(
-        os.path.join(RESULT_DIR, FQ2FA_OUT1), SAMPLE_S3_OUTPUT_PATH + "/")
+        fq2fa(input_fqs[1], result_path(FQ2FA_OUT2))
+    write_to_log("Finished FASTQ to FASTA conversion.")
+
+    dst = SAMPLE_S3_OUTPUT_PATH + "/"
+    uploader_start(result_path(FQ2FA_OUT1), dst)
     if len(input_fqs) == 2:
-        uploader_start(
-            os.path.join(RESULT_DIR, FQ2FA_OUT2), SAMPLE_S3_OUTPUT_PATH + "/")
+        uploader_start(result_path(FQ2FA_OUT2), dst)
 
 
 def run_cdhitdup(input_fas, uploader_start):
+    """CD-HIT-DUP is used to identify duplicates from single or paired reads.
+    Two FASTQ inputs means paired reads.
+
+    See: http://weizhongli-lab.org/cd-hit/
+    """
     cdhitdup_params = [
-        CDHITDUP, '-i', input_fas[0], '-o', RESULT_DIR + '/' + CDHITDUP_OUT1,
+        CDHITDUP, '-i', input_fas[0], '-o', result_path(CDHITDUP_OUT1),
         '-e', '0.05', '-u', '70'
     ]
     if len(input_fas) == 2:
         cdhitdup_params.extend(
-            ['-i2', input_fas[1], '-o2', RESULT_DIR + '/' + CDHITDUP_OUT2])
+            ['-i2', input_fas[1], '-o2', result_path(CDHITDUP_OUT2)])
     execute_command(" ".join(cdhitdup_params))
-    uploader_start(
-        os.path.join(RESULT_DIR, CDHITDUP_OUT1), SAMPLE_S3_OUTPUT_PATH + "/")
+
+    dst = SAMPLE_S3_OUTPUT_PATH + "/"
+    uploader_start(result_path(CDHITDUP_OUT1), dst)
     if len(input_fas) == 2:
-        uploader_start(
-            os.path.join(RESULT_DIR, CDHITDUP_OUT2),
-            SAMPLE_S3_OUTPUT_PATH + "/")
-    write_to_log("finished job")
+        uploader_start(result_path(CDHITDUP_OUT2), dst)
+    print("Finished CD-HIT-DUP.")
 
 
 def run_lzw(input_fas, uploader_start):
-    output_prefix = RESULT_DIR + '/' + LZW_OUT1[:-8]
+    output_prefix = result_path(LZW_OUT1[:-8])
     if len(input_fas) == 2:
         generate_lzw_filtered_paired(input_fas[0], input_fas[1], output_prefix,
                                      LZW_FRACTION_CUTOFFS)
     else:
         generate_lzw_filtered_single(input_fas[0], output_prefix,
                                      LZW_FRACTION_CUTOFFS)
-    # copy back to aws
-    uploader_start(
-        os.path.join(RESULT_DIR, LZW_OUT1), SAMPLE_S3_OUTPUT_PATH + "/")
+    # Copy back to aws
+    dst = SAMPLE_S3_OUTPUT_PATH + "/"
+    uploader_start(result_path(LZW_OUT1), dst)
     if len(input_fas) == 2:
-        uploader_start(
-            os.path.join(RESULT_DIR, LZW_OUT2), SAMPLE_S3_OUTPUT_PATH + "/")
-    write_to_log("finished job")
+        uploader_start(result_path(LZW_OUT2), dst)
+    print("Finished running LZW.")
 
 
 def run_bowtie2(input_fas, uploader_start):
-    # check if genome downloaded already
+    """Bowtie is an aligner used for filtering out host genomes. Two input
```

Comment:
Love all the clarifying comments regarding external tools. I would change the first sentence of this one to "Bowtie2 is an aligner we use for filtering out reads that map to the host genome"